lib/commands/fleet/index.ts

```diff
@@ -66,7 +73,10 @@ export default class FleetCmd extends Command {
 
 		const { getApplication } = await import('../../utils/sdk');
 
-		const application = (await getApplication(getBalenaSdk(), params.fleet, {
+		const balena = getBalenaSdk();
+
+		const application = (await getApplication(balena, params.fleet, {
+			$select: 'id',
```

Comment:
This will break the rest the command for the case that `--view` isn't used. (The app_name etc will not be fetched) 
You can either not pass the `$select` in all cases, or just pass only a `$select` & no `$expand` when `--view` is used.
I'm atm in favor of the fist, more universal approach, since it's not big amount of data anyway,